mithril-common/src/entities/cardano_network.rs

```diff
@@ -74,6 +74,18 @@ impl CardanoNetwork {
             CardanoNetwork::TestNet(magic_id) => magic_id,
         }
     }
+
+    /// Evaluate the possibility of validating the request to allow unparsable block or not,
+    /// depending on the target Cardano network
```

Comment:
Can you rewrite this one so it's not clear enough IMO?

Also we should probably rename the function to `compute_allow_unparsable_block`?

Reply:
Done, is the [comment](https://github.com/input-output-hk/mithril/blob/7467237f72ccd70c012f80f2eeead7621fa1f70c/mithril-common/src/entities/cardano_network.rs#L78) clear enough?